ui-tests/tests/timeline-slider.spec.ts

```diff
@@ -23,17 +23,51 @@ async function openNotebook(page: Page, notebookPath: string) {
   await page.waitForSelector('.jp-Notebook', { state: 'visible' });
 }
 
-test.describe('Open from Path', () => {
 
-  test('should fail if there are console errors', async ({ page, tmpPath }) => {
+const isTimelineEnv = process.env.TIMELINE_FEATURE || "0";
+const isTimeline = parseInt(isTimelineEnv)
+
+test.describe('Timeline Slider', () => {
+
+  isTimeline && test.use({ autoGoto: false })
```

Comment:
What is it for?

Reply:
So I found that even with different configuration, somehow playwright overwrites the baseurl and goes to the default one (`http://localhost:8888`) by default so we need to use this `test.use({ autoGoto: false })`

Reply:
isTimeline is the environment variable used to determine whether we are testing with the new environment or the default one. So we need to do this `test.use({ autoGoto: false })` only if we are testing with the new env (when `isTimeline` is true like in here `TIMELINE_FEATURE=1 jlpm test:timeline` )

Reply:
Could you explain what you mean by "new environment" and "default environment"?

Reply:
The "new environment" refers to the testing setup, which uses a custom `baseUrl` (`/api/collaboration/timeline/`) as opposed to the "default environment," where tests run with the standard `baseUrl` (`/lab` or `/`). The new environment is specifically configured to validate the Timeline functionality in isolation, whereas the default environment is used for general tests without the Timeline-specific context.

`isTimeline` is used to identify which environment is active. If `isTimeline` is true, we disable the automatic navigation (`test.use({ autoGoto: false })`) that Playwright performs, as this navigation doesn’t respect the custom `baseUrl `and defaults to `http://localhost:8888`. Disabling it allows us to manually control the navigation to the appropriate custom environment.